camunda-commons-ui/lib/services/index.js

```diff
@@ -97,7 +97,7 @@ ngModule.config([
     var window = $windowProvider.$get();
     var uri = window.location.href;
 
-    var match = uri.match(/\/app\/([\w-]+)\/([\w-]+)\//);
+    var match = uri.match(/\/(?:app)(?!.*app)\/([\w-]+)\/([\w-]+)\//);
```

Comment:
Let's add the `/` to the negative lookahead so it does not match query parameters or partial path names (e.g. .../app/cockpit/default/#/process-definition/*app*leOrder)

```suggestion
    var match = uri.match(/\/(?:app)(?!.*\/app\/)\/([\w-]+)\/([\w-]+)\//);
```